woocommerce/woocommerce-gutenberg-products-block
# Remove `Approved Pull Requests` Workflow

## What

This removes the `Approved Pull Requests` workflow, which adds the 'status: ready to merge' label, removes the `status: needs review` label, and adds the PR to the latest milestone.

These jobs will no longer be relevant once this repository is moved into the monorepo.

Fixes #

## Why


See internal conversation for context: pdToLP-ZC-p2#comment-745

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

_Note: this can only be tested once it has been merged, since it pertains to a PR workflow action._

1. Approve a PR (once this PR has been merged, as noted above).
2. Confirm no `status: ready to merge` label is created/added.
3. Confirm nothing is done with the `status: needs review` label, if present.
4. Confirm the PR is not added to the latest milestone.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
